recipes/bolt/meta.yaml

```diff
@@ -0,0 +1,119 @@
+{% set version = "17.0.6" %}
+{% set major_ver = version.split(".")[0] %}
+
+# as of LLVM 17, we expect an "rc" suffix for the release candidates
+{% set extra = "rc" if version.split(".")[-1] not in "0123456789" else "" %}
+{% set extra = "git" if version.split(".")[-1] == "dev0" else extra %}
+
+package:
+  name: bolt-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/llvm/llvm-project/releases/download/llvmorg-{{ version.replace(".rc", "-rc") }}/llvm-project-{{ version.replace(".rc", "rc") }}.src.tar.xz
+  sha256: 58a8818c60e6627064f312dbf46c02d9949956558340938b71cf731ad8bc0813
+  # matching https://github.com/conda-forge/llvmdev-feedstock/tree/main/recipe/patches,
+  # since we rebuild llvmdev here (until https://github.com/llvm/llvm-project/pull/97130)
+  patches:
+    - patches/0001-pass-through-QEMU_LD_PREFIX-SDKROOT.patch
+    # backports patch for issues with LLJIT, see
+    # https://github.com/llvm/llvm-project/commit/122ebe3b500190b1f408e2e6db753853e297ba28
+    - patches/0002-ORC-Use-EPC-bootstrap-symbols-to-communicate-eh-fram.patch
+
+build:
+  number: 0
+  merge_build_host: false
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+    - libcxx {{ cxx_compiler_version }}  # [osx]
+  host:
+    - libcxx {{ cxx_compiler_version }}  # [osx]
+    # needs aarch/ppc/arm (re)build of conda-forge/backtrace-feedstock
+    - backtrace                          # [unix and x86]
+    - libxml2
+    - zlib
+    - zstd
+
+outputs:
+  # Contains bolt support libraries
+  - name: libbolt
```

Comment:
This should be `libbolt-devel`. `libbolt` should be for the shared libraries.

Reply:
Bolt doesn't seem to have shared libraries at all, but happy to rename.